### PR TITLE
Only Author Can Delete A Question

### LINKED ISSUE #175: Only Author can Delete Questions

### PR DESCRIPTION
**Pull request by carson-ricca**
- Tested functionality and it works. Only the author of the question is allowed to delete it.
- Closes #175

### CODE REVIEW

File: api/permissions.py [hunk 1]
```diff
@@ -17,3 +17,10 @@ def has_permission(self, request, view):
 
     def has_object_permission(self, request, view, obj):
         return request.user == obj.user
+
+
+class IsOwnerOrReadOnly(permissions.IsAuthenticated):
```
Comment (keyvankhademi) [inline api/permissions.py:22]: We want them to be able to edit it but not delete it.

**Comment by keyvankhademi** [comment]
This also has a UI task to disable the delete button if they are not the author.

File: api/permissions.py (hunk 1 above)
Comment (carson-ricca) [reply api/permissions.py:22]: Oh whoops I forgot this disabled that.

**Comment by carson-ricca** [comment]
> This also has a UI task to disable the delete button if they are not the author.

Will fix this tomorrow.

**Review by keyvankhademi — APPROVED** [review]

**Pull request merged**